### PR TITLE
add rule for airelf.com.tw

### PR DESCRIPTION
**Pull request by PeterDaveHello**

### CODE REVIEW

**Comment by J0WI** [comment]
I also found `168.airelf.com.tw` and `chat.airelf.com.tw`.

File: src/chrome/content/rules/airelf.com.tw.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<ruleset name="airelf.com.tw">
+	<target host="airelf.com.tw" />
+	<target host="www.airelf.com.tw" />
+
+	<securecookie host="^\.airelf\.com\.tw$" name=".+" />
+
+	<rule from="^http://airelf\.com\.tw/"
+        to="https://www.airelf.com.tw/" />
```
Comment (J0WI) [inline src/chrome/content/rules/airelf.com.tw.xml]: Please document why this is needed.

Comment (PeterDaveHello) [inline src/chrome/content/rules/airelf.com.tw.xml]: `https://airelf.com.tw` will get 400 error.

Comment (J0WI) [inline src/chrome/content/rules/airelf.com.tw.xml]: Yes, but please use a comment inside the rule for documentation. ;)

Comment (PeterDaveHello) [inline src/chrome/content/rules/airelf.com.tw.xml]: Okay, updated.

**Comment by J0WI** [comment]
> I also found 168.airelf.com.tw and chat.airelf.com.tw.

Please add a comment for the mismatch on `chat.airelf.com.tw` as well.

**Comment by PeterDaveHello** [comment]
Okay, updated.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**